components/activity_and_insights/insights/insights.tsx

```diff
@@ -72,7 +72,7 @@ const Insights = () => {
             LocalStorageStore.setPreviousViewedType(currentUserId, currentTeamId, PreviousViewedTypes.INSIGHTS);
         }
 
-        if (isStarterFree) {
+        if (isStarterFree || !isEnterpriseReady) {
```

Comment:
unrelated to this change, but i just noticed this useEffect doesnt have any dependency, is this meant to run on any changes?